### PR TITLE
Refactor configurations

### PR DESCRIPTION
**Pull request by fabianoroberto**

### CODE REVIEW

File: Entity/Media.php [hunk 1]
```diff
@@ -373,17 +375,17 @@ public function getFilters()
     }
 
     /**
-     * Clear all filters
+     * Clear all filters.
```
Comment (LuigiCardamone) [inline Entity/Media.php:313]: This dot on every comment is useless. We already disabled this on other projects

File: Entity/Media/CropInfo.php [hunk 2]
```diff
@@ -1,54 +1,55 @@
...
 
     /**
-     * @var integer rotation degree
+     * @var int rotation degree
      */
     private $rotation;
 
     /**
-     * @var integer priority in gallery
+     * @var int priority in gallery
```
Comment (LuigiCardamone) [inline Entity/Media/CropInfo.php]: Why gallery priority inside a class called CropInfo?

Comment (rollsappletree) [reply Entity/Media/CropInfo.php]: Agree

Comment (fabianoroberto) [reply Entity/Media/CropInfo.php]: Outdated

**Pull request merged**